editor/modules/aariba/behaviour.ts

```diff
@@ -21,7 +21,7 @@ let SAFE_INSERT_BEFORE_TOKENS =
 // everything except the '{' and then adding again a '\n'
 // does not behave identically. Furthermore, I think we should
 // match the brackets. (Lookup for the next possible bracket).
-let context;
+let context: any;
```

Comment:
You’ve got a lot of `any` in you code. Do you plan to (*can you*) be more specific in the future?